src/components/pages/career-details/new-career-form/career-form.tsx

```diff
@@ -0,0 +1,324 @@
+          <TextInput
+            name="last_name"
+            label={t<string>('career.last-name')}
+            control={control}
+            rules={{ required: t<string>('career.error.last-name') }}
+          />
+
+          <TextInput
+            name="email"
+            label={t<string>('career.email')}
+            control={control}
+            rules={{
+              required: t<string>('career.error.email'),
+              pattern: {
+                value: SIMPLE_EMAIL_PATTERN,
+                message: t<string>('career.error.email-undefined'),
+              },
+            }}
+          />
+          <TextInput
+            name="location"
+            label={t<string>('career.location')}
+            control={control}
+          />
+          <TextInput
+            name="available_from"
+            label={t<string>('career.available-from')}
+            control={control}
+          />
+          <TextInput
+            name="salary_expectations"
+            label={t<string>('career.salary-expectations')}
+            control={control}
+          />
+        </FormLayout>
+        <TextArea
+          name="message"
+          label={t<string>('career.cover-letter')}
+          control={control}
+        />
+
+        <br />
+        <FileDropper
+          register={register}
+          setValue={setValue}
+          setError={setError}
+          clearErrors={clearErrors}
+          errors={errors}
+          name="documents"
+          fileCategories={fileCategories}
+          acceptedFileTypes={'.pdf'}
+          illustration="monitor_024"
+          maxFiles={3}
+          validator={validator}
+        />
+
+        <TextWrapper>
+          <Trans i18nKey={'career.info-text'} />
+        </TextWrapper>
+        <br />
+        <Checkbox
+          name="privacy"
+          label={
+            <Trans i18nKey={'career.privacy-policy'}>
+              <span>
+                Hiermit bestätige ich, dass ich die
+                <StyledLink to={PRIVACY_POLICY}>
+                  Datenschutzerklärung
+                </StyledLink>
+                zur Kenntnis genommen habe
+              </span>
+            </Trans>
+          }
+          control={control}
+          rules={{ required: t<string>('career.error.approval') }}
+        />
+        <br />
+        <Button type="submit" disabled={isSubmitting}>
+          {!errors?.api
+            ? t<string>('career.action.send')
+            : t<string>('career.action.again')}
+        </Button>
+        {errors?.api && (
+          <StyledErrorMessage>
+            <Trans id="career.action.again-text">
+              <span>
+                Versuch es bitte noch einmal. Klappt es nicht dann schicke deine
+                Bewerbung direkt an{' '}
+                <StyledLink to="mailto:career@satellytes.com">
+                  career@satellytes.com
+                </StyledLink>
+              </span>
+            </Trans>
+          </StyledErrorMessage>
+        )}
```

Comment:
This could also be a local component `CareerFormErrors` or such.

Reply:
👍

Reply:
seen that, thanks!